towhee/serve/triton/dockerfiles/DockerfileCuda114

```diff
@@ -27,6 +27,10 @@ RUN apt-get install git-lfs && \
 
 WORKDIR /workspace
 
+RUN git clone -b nebullvm https://github.com/towhee-io/towhee-compiler.git
```

Comment:
Why not build a pip package, and use pip to install it.

Reply:
@reiase Can we release a version base on this branch?